airflow/providers/papermill/example_dags/example_papermill.py

```diff
@@ -82,10 +83,18 @@ def check_notebook(inlets, execution_date):
         parameters={"msgs": "Ran from Airflow at {{ execution_date }}!"}
     )
 
-    check_output = PythonOperator(
-        task_id='check_out',
-        python_callable=check_notebook,
-        inlets=AUTO
-    )
+    if version.startswith("1."):
```

Comment:
Rather than a version check I'd prefer we used "duck-typing" - i.e

```python
try:
    from airflow.lineage import AUTO
except ImportError:
    AUTO = None


...

if AUTO is not None:
    ...
```

Also: we are still doing `from airflow.lineage import AUTO` which will fail on 1.10, so this check is pointless as it stands :)

Reply:
It won't fail. Lineage is copied to "papermill" provider and Bowler refactors it to import it from there. It imports nicely.

Reply:
That seems odd? Lineage is part of core, isn't it? Does the lineage actually work like this? Nothing inside papermill_operator.py looks at lineage, so I don't see why we need to include it in the backport.

Is this new in rc3/this PR, or can I see this behaviour in Rc2 too?

This seems a higher risk approach then updating an example to remove lineage=AUTO to me.

Reply:
Sure I will remove this lineage import and fallback this way. That's even easier for me

Reply:
Actually the problem is that Papermilll is all about lineage, It imports NoteBook and File from lineage package and that's why I implemented those workarounds. But yes. Let's simply remove it altogether. It makes no sense with all those hacks.